extra.js

```diff
@@ -10,10 +10,15 @@
  * The numbers should always be positive */
 
 function calculator(operator, a, b) {
-  switch (
-    operator
-    // CODE HERE
-  ) {
+  switch (operator) {
+    case "+":
+      return a+b;
+    case "-":
+      return a-b;
+    case "/":
+      return a/b;
+    case "*":
+      return a*b;
   }
 }
 

```

Comment:
use "if statement" to run the switch case only when the inputs(a & b) are positive